script/ci-release-build.js

```diff
@@ -14,7 +14,9 @@ const circleCIJobs = [
   'linux-arm-publish',
   'linux-arm64-publish',
   'linux-ia32-publish',
-  'linux-x64-publish'
+  'linux-x64-publish',
+  'mas-publish',
+  'osx-publish'
 ]
 
 const vstsJobs = [
```

Comment:
There's another commit somewhere that removes the VSTS jobs from this file, we need that commit as well

Reply:
https://github.com/electron/electron/commit/a569dad8c52303e292597c9005ec470eb97d13bc

Reply:
Good catch @MarshallOfSound.  Cherry pick on its way.